<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CascadeControls</class>
 <widget class="QWidget" name="CascadeControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QSpinBox" name="timeChannelBox">
       <property name="specialValueText">
        <string>all</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="foilLabel">
       <property name="text">
        <string>Foil</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="foilBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="singleSlidesBox">
       <property name="text">
        <string>Single Slides</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="timeChannelLabel">
       <property name="text">
        <string>Time channel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>singleSlidesBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>foilBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleSlidesBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeChannelBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleSlidesBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>foilLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleSlidesBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeChannelLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
